.github/ISSUE_TEMPLATE/e2e_test_report.md

```diff
@@ -0,0 +1,36 @@
+---
+name: e2e test report
+about: Report an e2e test failure.
+title: ''
+labels: ['category: e2e', 'needs triage']
+assignees: ''
+
+---
+
+### Description
+<!-- Before reporting an issue, are you sure this e2e test is failing consistently? -->
+<!-- Add a clear and concise description of what the issue is. Please, be as descriptive as possible. -->
+
+**Error**:
+<!-- The error that the e2e test indicates. -->
+
+**Spec**:
+<!-- The test file that's triggering this error. -->
+
+**Relevant line(s) of code**:
+<!-- Line of code that is triggering this error. -->
```

Comment:
Is this trivial? I like that there are prompts here to capture the relevant information. but at the same time it's important to make this a zero-minute frictionless process.

If the test failures are visible in GitHub actions logs, maybe it's extra bureaucracy to require reporter to extract specific details and paste here?

TLDR I think the prompts are useful but we should be lenient on what we expect. 

Some options:
- Ask the reporter to paste the relevant chunk of the output from the test failure. This means we expect them to at least look at the logs, and paste in a chunk that covers the prompts above.

Reply:
Good call out! I've adopted your suggestion and changed it in https://github.com/Automattic/woocommerce-payments/pull/8420/commits/e6d7a0276b8b9e11c6bd7f7d2d94e12f0a9d2108.